pkg/registry/registry_to_model_test.go

```diff
@@ -0,0 +1,71 @@
+package registry
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/operator-framework/operator-registry/internal/model"
+	"github.com/operator-framework/operator-registry/internal/property"
+	"github.com/operator-framework/operator-registry/pkg/image"
+)
+
+func TestConvertRegistryBundleToModelBundle(t *testing.T) {
+	registryBundle, err := testRegistryBundle()
+	assert.NoError(t, err)
+	expected := testModelBundle()
+
+	actual, err := registryBundleToModelBundle(registryBundle)
+	require.NoError(t, err)
+	assertEqualsModelBundle(t, expected, *actual)
+
+	registryBundles, err := ConvertRegistryBundleToModelBundles(registryBundle)
+	assert.Equal(t, len(registryBundles), 2)
+}
+
+func testModelBundle() model.Bundle {
+	b := model.Bundle{
+		Name:     "etcdoperator.v0.9.2",
+		Image:    "quay.io/operatorhubio/etcd:v0.9.2",
+		Replaces: "etcdoperator.v0.9.0",
+		Skips:    []string{"etcdoperator.v0.9.1"},
+		Properties: []property.Property{
+			property.MustBuildChannel("alpha", "etcdoperator.v0.9.0"),
+			property.MustBuildChannel("stable", "etcdoperator.v0.9.0"),
+			property.MustBuildPackage("etcd", "0.9.2"),
+			property.MustBuildSkips("etcdoperator.v0.9.1"),
+			//TODO(anik120): check if etcdclusters. is supposed to be prefixed
+			property.MustBuildGVKRequired("etcd.database.coreos.com", "v1beta2", "EtcdCluster"),
+			property.MustBuildGVKRequired("testapi.coreos.com", "v1", "testapi"),
```

Comment:
Not sure if it matters in this test, but kinds are always uppercased.
```suggestion
			property.MustBuildGVKRequired("testapi.coreos.com", "v1", "Testapi"),
```